drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -139,7 +139,12 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x006F
     deviceProfileName: plug-binary
-
+#Legrand
+  - id: "4129/3"
+    deviceLabel: Smart Lights Smart Plug
```

Comment:
Just confirming this isn't meant to be "Smart Light Plug" or something like that, right?

Reply:
I verified   this is what the partner submitted, so I think this is intentional.